web/tests/recent_topics.test.js

```diff
@@ -88,7 +90,12 @@ mock_esm("../src/message_list_data", {
     MessageListData: class {},
 });
 mock_esm("../src/message_store", {
-    get: (msg_id) => messages[msg_id - 1],
+    get(msg_id) {
+        if (msg_id < 12) {
+            return messages[msg_id - 1];
+        }
+        return private_messages[msg_id - 12];
+    },
```

Comment:
Putting the new messages in `messages` will make other tests to fail. Because in many tests we are processing all the messages inside `messages`.
```
web/tests/recent_topics.test.js:427:    rt.process_messages(messages);
web/tests/recent_topics.test.js:587:    rt.process_messages(messages);
web/tests/recent_topics.test.js:708:    rt.process_messages(messages);
web/tests/recent_topics.test.js:789:    rt.process_messages(messages);
web/tests/recent_topics.test.js:809:    rt.process_messages(messages);
web/tests/recent_topics.test.js:937:    rt.process_messages(messages);
web/tests/recent_topics.test.js:986:    rt.process_messages(messages);
web/tests/recent_topics.test.js:1028:    rt.process_messages(messages);
```
So , I thought it will be better to create a new variable for private conversations instead of changing existing tests.

Reply:
I think it'd probably be better long-term to add them to `messages`, and just extend the verification in those existing tests to expect additional content.

If it takes more than an hour to do, we can potentially decide to merge this for now and plan that as a follow-up project.